crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -209,6 +209,16 @@ pub struct NetworkConfig {
     #[schemars(schema_with = "holochain_util::jsonschema::url2_schema")]
     pub signal_url: url2::Url2,
 
+    /// The interval between initiating gossip rounds.
+    pub gossip_initiate_interval_ms: Option<u32>,
+
+    /// The minimum amount of time that must be allowed to pass before a gossip round can be
+    /// initiated by a given peer.
+    pub gossip_min_initiate_interval_ms: Option<u32>,
+
+    /// The timeout for a gossip round.
+    pub gossip_round_timeout_ms: Option<u32>,
```

Comment:
I'm worried about this for (1) the churn as we make changes to kitsune I'd prefer to not have to update the conductor api / config every time we want to bump the version and (2) because I don't think anyone using holochain should change these settings ever.

Would it not work to just apply configs like these to the advanced field directly? We could have a `#[cfg(feature = "test_utils")]` fn helper that applies some things like these for testing, and then on bumping kitsune version we'd only have to update the body of that function, and not every single callsite?

Reply:
Looking better? https://github.com/holochain/holochain/pull/4787/commits/4b72fb508192aa132b0f05d007698582703cb3d2

Reply:
Yep, thanks! :guitar: